test/functionalTest/cases/715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test

```diff
@@ -0,0 +1,216 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Maybe we should add a a .test to ensure that X-Auth-Token is not being forwarded when X-Auth-Token is not included in the request? Or existing tests are covering that functionality?